addons/l10n_jo_edi/models/account_edi_xml_ubl_21_jo.py

```diff
@@ -60,7 +60,8 @@ def _get_line_taxable_amount(self, line):
         return self._round_max_dp(self._get_unit_price_jod(line)) * self._round_max_dp(line.quantity) - self._round_max_dp(self._get_line_discount_jod(line))
 
     def _get_payment_method_code(self, invoice):
-        return PAYMENT_CODES_MAP[invoice.company_id.l10n_jo_edi_taxpayer_type]['receivable']
+        taxpayer_type = invoice.company_id.l10n_jo_edi_taxpayer_type
+        return PAYMENT_CODES_MAP.get(taxpayer_type, {}).get('receivable', '')
```

Comment:
```suggestion
        return PAYMENT_CODES_MAP.get(invoice.company_id.l10n_jo_edi_taxpayer_type, {}).get('receivable', '')
```
We can keep the same code as in the fix already merged.